querybook/webapp/ui/Link/Link.tsx

```diff
@@ -27,6 +27,17 @@ function isInternalUrl(url: LocationDescriptor): boolean {
     return false;
 }
 
+function getExternalUrl(url: LocationDescriptor) {
```

Comment:
for external url, whether it starts with `https?` or not, I think it's fine to put it in the `href`.  like `<a href="google.com">google</a>`

or are you trying to solve any other issue?

Reply:
that example you put would not redirect to a new site, https://stackoverflow.com/questions/43803778/href-without-https-prefix

Reply:
you're right. seems prepending `//` is a better option. also can we merge the check as 
```
if (url && typeof url === 'string' && !/^[a-zA-Z]+:\/\//.test(url)) {
  return `//${url}`
}

return url
```